api/zpoller.api

```diff
@@ -55,7 +55,7 @@
         timeout expired, returns NULL. You can test the actual exit condition by 
         calling zpoller_expired () and zpoller_terminated (). The timeout is in 
         msec.
-        <argument name = "timeout" type = "integer" />
+        <argument name = "timeout" type = "msecs" />
```

Comment:
this is a stable api, signature cannot change

Reply:
There are inconsistencies in the API causing (a lot of) compilation warnings in clang. The proposed modifications do not introduce incompatibilities and enable to remove the warnings. Basically, we move from int to int64_t, which should always work.

Reply:
Please see https://rfc.zeromq.org/spec/44/ - we do not do API changes after a class is declared stable, sorry

Reply:
In this case, the proposed changes are to avoid actual possible data loss in the existing code. Warnings are generated for a reason. But I get it. The rules are the rules :-)